app/assets/javascripts/validation.coffee

```diff
@@ -143,7 +143,11 @@ $(document).ready ->
         else
           "#{field} should have one type completed"
       when 'invalidPicklist' then "#{field} #{error.message}"
+      when 'enum' then "#{field} value [#{$('#' + error.id + ' option:selected').text()}] does not match a valid selection option"
       when 'anyOf' then "#{error.message}"
+      # UseConstraintsType is the only place a 'not' validation is used
+      # so this is a very specific message
+      when 'not' then 'License Url and License Text cannot be used together'
```

Comment:
I can only trigger this validation error on `submit`, not on `blur` - should we see it on `blur`? Also, I feel like we should have a test to trigger this error since it is unusual and the only `not` in the schema. I can check this in the new branch though.